classes/fields/datetime.php

```diff
@@ -181,6 +181,15 @@ public function options() {
 					'hh_mm_ss' => date_i18n( 'H:i:s' ),
 				),
 			),
+			static::$type . '_year_range'      => array(
+				'label'   => __( 'Year range', 'pods' ),
+				'default' => '',
+				'type'    => 'text',
+				'help'    => sprintf(
```

Comment:
Can we show an example of at least 1-2 options in this help text? This isn't super helpful if we just have a link here.